### ISSUE TITLE
integrate visualization panes

### ISSUE THREAD

**Issue by kodonnell** [nz-stefan/shiny_ufo_app#2]
e.g. when I zoom to an area on the map, the other panes (and dropdowns etc.) should change to being filtered by that area. Or if I click a country in the bar graph, the map zooms there.

**Comment by nz-stefan** [comment]
Yep, good idea. Will see what events I can capture from charts and map.
